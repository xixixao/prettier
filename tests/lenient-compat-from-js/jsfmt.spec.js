run_spec(__dirname, ["flow"], {
  bracketSpacing: false,
  lenient: true,
  lenientCompat: true,
  semi: false
});
