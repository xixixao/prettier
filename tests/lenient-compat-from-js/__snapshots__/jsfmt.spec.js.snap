// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`arrows.js - flow-verify 1`] = `
ob.forEach(x => ({
  a: "hey",
}));

const h = () => {};
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
ob.forEach(x => ({
  a: "hey"
}))

const h = () => {}

`;
